src/api/goods.ts

```diff
@@ -10,10 +10,10 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => [...goods].sort().slice(0, 5));
```

Comment:
`good` is an object, so you can't just do `.sort()`
add callback in the `sort` method